test/unit/math/mix/fun/atan2_test.cpp

```diff
@@ -16,3 +16,18 @@ TEST(mathMixCore, atan2) {
   stan::test::expect_ad(f, 1.2, 3.9);
   stan::test::expect_ad(f, 0.5, 2.3);
 }
+/* Not sure how to test this so I'll come back to it.
```

Comment:
Now iz the time

Reply:
This is only for unary stuff so another time